package.json

```diff
@@ -12,16 +12,12 @@
     "@types/jest": "^24.9.1",
     "jest": "^24.9.0",
     "lerna": "^3.20.2",
-    "npm-force-resolutions": "^0.0.3",
     "rimraf": "~3.0.0",
     "ts-jest": "^24.3.0",
     "typescript": "~3.7.3",
     "minimist": ">=1.2.3"
   },
   "engines": {
     "node": ">=10"
-  },
-  "resolutions": {
```

Comment:
previously there is a type conflict so we add this resolution explicitly, now there is no error in compilation after upgrading these packages, right?

Reply:
Correct, this builds out of the box now.

In fact this override was preventing the build from working correctly.